packages/webdriverio/src/commands/browser/react$.js

```diff
@@ -20,9 +20,10 @@
  * </example>
  *
  * @alias browser.react$
+ * @param {String=} selector of React component
  * @param {Object=} props  React props the element should contain
  * @param {Object=} state  React state the element should be in
- * @return {Element}
+ * @return {Element[]}
```

Comment:
not sure why this change
```suggestion
 * @return {Element}
```